galicaster/mediapackage/mediapackage.py

```diff
@@ -23,6 +23,8 @@
 import re
 import time
 import os
+import dateutil
```

Comment:
It's not possible to `import dateutil` and use `dateutil.parser.parse()`

http://stackoverflow.com/a/23385208